### ISSUE TITLE
training everything good, validing mAP 0

### ISSUE THREAD

**Issue by leetesua** [open-mmlab/mmdetection#8645]
![871dda13734707376fa7e79a710f10a](https://user-images.githubusercontent.com/25045281/186887513-86430f21-d834-4a1c-acf5-49fcaf1c6112.png)
-------------------------------------------------------
I trained my coco-style custom dataset on DETR. Everything looks good on training phase. But always got 0 during evaluation. All metrics results are gross eggs.
I also trained the same data and using the same setting on Faster rcnn & YOLOX. And it went well. So, maybe something wrong with Detr?

**Comment by leetesua** [comment]
Thanks in advance if anyone has a clue!

**Comment by BIGWangYuDong** [comment]
You can look at your log file, whether there is a strange loss while training. Also, maybe you can show your result to check whether DETR has detected anything. Seems this is some training error during training DETER

**Comment by Chop1** [comment]
Transformer are very hard to train. They require lots of data and long training shedule.
If you look at the paper, they need ~100 epoch on COCO before having some good detection...

I strongly suggest to use a pretrained model. I'm pretty sure it will solve your problem. (I observe the same thing; without pretraining map is 0, with pretraining it works.)
